features/pre.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <pre>
-description: The `<pre>` element TODO.
+description: The `<pre>` element epresents preformatted text which is to be presented exactly as written in the HTML file.
```

```suggestion
description: The `<pre>` element represents preformatted text which is to be presented exactly as written in the HTML file.
```

I wonder if it would be better to focus on how whitespace is preserved rather than saying that it's exactly as written. Unlike `<xmp>`, the normal rules of parsing aren't suspended inside `<pre>`.